src/test/java/com/scriptbasic/lexer/TestBasicLexicalAnalyzer.java

```diff
@@ -101,6 +101,12 @@ public void newLineIsAnalyzedAs_surprise_surprise_newLine() throws AnalysisExcep
         );
     }
 
+    @Test
+    public void colonIsAnalyzedAs_statementSeparator() throws AnalysisException {
+        assertLexicals(from(":"), SYMBOL(":")
+        );
```

Comment:
why do we need this closing paren on a separate line? Please put it at the end of the previous line.

Reply:
Same formatting was used in other tests in this file. I did it same way.
Now I changed the formatting of other tests.